andela/ah-the-phoenix-frontend
# #163388099 Get and Display User Profile

**What does this PR do?**

-Implement user view and edit their profile

**Description of Task to be completed?**

-A user can view their profile
-A user can edit their profile
**How should this be manually tested?**

-Clone the github repository git clone https://github.com/andela/ah-the-phoenix-frontend on your local machine.
-Change your working directory to the project folder cd ah-fulldeck-frontend
-Checkout to thebranch ft-user-profile-163388099 using the command git checkout ft-view-edit-profile-163388099
Install the required dependencies using npm install
Run the local development server using npm start and you should get the following message:
Go to the link /profiles/ where the username is your registered username
**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
#163388099
